python/servo/platform/base.py

```diff
@@ -68,6 +69,14 @@ def bootstrap(self, force: bool, skip_platform: bool, skip_lints: bool):
         if not installed_something:
             print("Dependencies were already installed!")
 
+    def install_rust_toolchain(self) -> bool:
```

Comment:
Should the return type annotation be removed?  the body only returns `None` or throws.

Reply:
oops! fixed in 7e81791652b3da5f8ab218e4920072711da90d81